tests/with-raw-glucose.test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var should = require('should');
+var withRawGlucose = require('../lib/with-raw-glucose');
+
+var cal = {
+  scale: 1
+  , intercept: 25717.82377004309
+  , slope: 766.895601715918
+};
+
+describe('IOB', function ( ) {
+  it('should add raw glucose and not mess with real glucose', function ( ) {
+    var entry = {unfiltered: 113680, filtered: 111232, glucose: 110, noise: 1};
+    withRawGlucose(entry, cal);
+
+    entry.glucose.should.equal(110);
+    entry.raw.should.equal(113);
+  });
+
+  it('should add raw glucose and set missing glucose', function ( ) {
+    var entry = {unfiltered: 113680, filtered: 111232, noise: 1};
+    withRawGlucose(entry, cal);
+
+    entry.glucose.should.equal(115);
+    entry.raw.should.equal(115);
+  });
+
+  it('should add raw glucose, but set missing glucose above maxRaw', function ( ) {
```

Comment:
Missing a not?  "it('should add raw glucose, but not set missing glucose above maxRaw'"?